modules/installation-azure-marketplace-subsribe.adoc

```diff
@@ -0,0 +1,116 @@
+// * installing/installing_aws/installing-azure-user-infra.adoc
+
+ifeval::["{context}" == "installing-azure-customizations"]
+:ipi:
+endif::[]
+ifeval::["{context}" == "installing-azure-user-infra"]
+:upi:
+endif::[]
+
+//mpytlak: The procedure differs depending on whether this module is used in an IPI or UPI assembly.
+
+:_content-type: PROCEDURE
+[id="installation-azure-marketplace-subscribe_{context}"]
+= Selecting an Azure Marketplace image
+If you are deploying an {product-title} cluster using the Azure Marketplace offering, you must first obtain the Azure Marketplace image. The installation program uses this image to deploy worker nodes.
+
+.Prerequisites
+
+* You have installed the Azure CLI client `(az)`.
+* Your Azure account is entitled for the offer and you have logged into this account with the Azure CLI client.
+
+[NOTE]
+=====
+How you locate your Azure Marketplace image is different depending on the region from which the offer was purchased. In North America, `redhat` is the publisher. In EMEA, `redhat-limited` is the publisher.
+=====
+
+.Procedure
+
+. Display all of the available {product-title} images by running one of the following commands:
++
+** North America:
++
+[source,terminal]
+----
+$  az vm image list --all --offer rh-ocp-worker --publisher redhat
+----
+** EMEA:
++
+[source,terminal]
+----
+$  az vm image list --all --offer rh-ocp-worker --publisher redhat-limited
+----
+. Inspect the image for your offer by running one of the following commands:
+** North America:
++
+[source,terminal]
+----
+$ az vm image show --urn redhat:rh-ocp-worker:rh-ocp-worker:<version>
+----
+** EMEA:
++
+[source,terminal]
+----
+$ az vm image show --urn redhat-limited:rh-ocp-worker:rh-ocp-worker:<version>
+----
+. Review the terms of the offer by running one of the following commands:
+** North America:
++
+[source,terminal]
+----
+$ az vm image terms show --urn redhat:rh-ocp-worker:rh-ocp-worker:<version>
+----
+** EMEA:
++
+[source,terminal]
+----
+$ az vm image terms show --urn redhat-limited:rh-ocp-worker:rh-ocp-worker:<version>
+----
+. Accept the terms of the offering by running one of the following commands:
+** North America:
++
+[source,terminal]
+----
+$ az vm image terms accept --urn redhat:rh-ocp-worker:rh-ocp-worker:<version>
+----
+** EMEA:
++
+[source,terminal]
+----
+$ az vm image terms accept --urn redhat-limited:rh-ocp-worker:rh-ocp-worker:<version>
+----
+ifdef::ipi[]
+. Record the image details of your offer. You must update the `compute` section in the `install-config.yaml` file with values for `publisher`, `offer`, `sku`, and `version` before deploying the cluster.
+endif::ipi[]
+ifdef::upi[]
+. Record the image details of your offer. If you use the Azure Resource Manager (ARM) template to deploy your worker nodes, you can update `storageProfile.imageReference` by deleting the `id` parameter and adding the `offer`, `publisher`, `sku`, and `version` parameters using the values from your offer.
```

Comment:
Just noticed there's no `sku` in the YAML clip

Reply:
Thanks @jeana-redhat. Good catch. Fixed it.